src/scss/grommet-core/_objects.quote.scss

```diff
@@ -1,6 +1,7 @@
 .quote {
   border-width: $inuit-base-spacing-unit;
   border-style: solid;
+  max-width: calc(100vw - #{$inuit-base-spacing-unit * 3});
```

Comment:
Can we instead make this 100%?

What is the 3 * base spacing unit accounting for?